pkgs/development/compilers/dotnet/combinePackages.nix

```
@@ -0,0 +1,20 @@
+{ packages }: 
```

since packages is no longer an attr with multiple attributes, how about we just make it a plain array?
```suggestion
packages:
```